src/db/tables/onerep_scans.ts

```diff
@@ -72,35 +57,58 @@ async function setOnerepManualScan(
     onerep_profile_id: onerepProfileId,
     onerep_scan_id: onerepScanId,
     onerep_scan_reason: "manual",
+    // @ts-ignore knex.fn.now() results in it being set to a date,
+    // even if it's not typed as a JS date object:
     created_at: knex.fn.now(),
   });
 }
 
 async function setOnerepScanResults(
```

Comment:
nit: perhaps the word `set` isn't the best.. it usually implies update rather than append / add. 
Same with `setOnerepManualScan`